setup.py

```diff
@@ -9,14 +9,14 @@
     long_descr = f.read()
 
 setup(
-    name='ctakes-client',
-    version='0.2.4',
+    name='ctakesclient',
+    version='1.0.0',
```

Comment:
Switching from 0 to 1 is always a mental blocker, but since this was a breaking change, it makes sense to me to increment the major version. But if you would prefer to avoid doing breaking change semantic versioning until we release "for real", I'm fine with that too.